pkg/controller/node_test.go

```diff
@@ -63,3 +63,36 @@ func TestNodeIsReady(t *testing.T) {
 		}
 	}
 }
+
+func TestNodeIsReadyWithoutReadyLabel(t *testing.T) {
```

Comment:
Can we make a more general `TestNodeIsReady` that also covers the `Unscheduable` case? Still, thanks a lot for this fix and provided unit test :smiley:

Reply:
@FxKu I just changed the TestNodeReady Test to be more generic, please have a look